code/game/objects/items/rogueweapons/ranged/gunpowder/blunderbuss.dm

```diff
@@ -0,0 +1,276 @@
+/obj/item/gun/ballistic/blunderbuss
+	name = "blunderbuss"
+	desc = "A gunpowder weapon that shoots a large number of pellets"
+	icon = 'icons/roguetown/weapons/32.dmi'
+	icon_state = "blunderbuss"
+	lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
+	item_state = "blunderbuss"
+	force = 10
+	force_wielded = 15
+	possible_item_intents = list(/datum/intent/mace/strike/wood)
+	gripped_intents = list(/datum/intent/shoot/blunderbuss, /datum/intent/arc/blunderbuss, INTENT_GENERIC)
+	gripspriteonmob = TRUE
+	internal_magazine = TRUE
+	experimental_inhand = FALSE
+	mag_type = /obj/item/ammo_box/magazine/internal/blunderbuss
+	wlength = WLENGTH_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
+	randomspread = 0
+	spread = 0
+	can_parry = TRUE
+	pin = /obj/item/firing_pin
+	minstr = 6
+	walking_stick = FALSE
+	cartridge_wording = "grapeshot"
+	load_sound = 'sound/foley/musketload.ogg'
+	fire_sound = "sound/arquebus/arquefire.ogg"
+	anvilrepair = /datum/skill/craft/blacksmithing
+	smeltresult = /obj/item/ingot/steel
+	bolt_type = BOLT_TYPE_NO_BOLT
+	casing_ejector = FALSE
+	//pickup_sound = 'sound/sheath_sounds/draw_from_holster.ogg'
+	//sheathe_sound = 'sound/sheath_sounds/put_back_to_holster.ogg'
+	slot_flags = ITEM_SLOT_BACK
+	damfactor = 1
+	var/reloaded = FALSE
+	var/load_time = 50
+	var/gunpowder = FALSE
+	var/obj/item/ramrod/myrod = null
+	var/spread_num = 10
+	var/gunchannel
+
+/obj/item/gun/ballistic/blunderbuss/getonmobprop(tag)
+	. = ..()
+	if(tag)
+		switch(tag)
+			if("onback")
+				return list("shrink" = 0.5,"sx" = -1,"sy" = 2,"nx" = 0,"ny" = 2,"wx" = 2,"wy" = 1,"ex" = 0,"ey" = 1,"nturn" = 0,"sturn" = 0,"wturn" = 70,"eturn" = 15,"nflip" = 1,"sflip" = 1,"wflip" = 1,"eflip" = 1,"northabove" = 1,"southabove" = 0,"eastabove" = 0,"westabove" = 0)
+
+/obj/item/gun/ballistic/blunderbuss/Initialize()
+	. = ..()
+	myrod = new /obj/item/ramrod(src)
+
+
+/obj/item/gun/ballistic/blunderbuss/shoot_live_shot(mob/living/user as mob|obj, pointblank = 0, mob/pbtarget = null, message = 1)
+	fire_sound = pick("sound/arquebus/arquefire.ogg", "sound/arquebus/arquefire2.ogg", "sound/arquebus/arquefire3.ogg",
+				"sound/arquebus/arquefire4.ogg", "sound/arquebus/arquefire5.ogg")
+	. = ..()
+
+/obj/item/gun/ballistic/blunderbuss/attack_right(mob/user)
+	if(user.get_active_held_item())
+		return
+	else
+		if(myrod)
+			playsound(src, "sound/items/sharpen_short1.ogg",  100)
+			to_chat(user, "<span class='warning'>I draw the ramrod from the [src]!</span>")
```

Comment:
Please spannify this with the span_warning helper. This goes for all the ones that follow.